bots/tests-scan

```diff
@@ -366,21 +351,16 @@ def update_status(api, revision, context, last, changes):
         return False
     return True
 
-def cockpit_tasks(api, update, contexts, repo, pull_data, pull_number, sha, amqp):
+def cockpit_tasks(api, update, branch_contexts, repo, pull_data, pull_number, sha, amqp, is_external):
```

Comment:
Why do we need this new option? Doesn't this just cement the asymmetry? When would it be different to `repo != "cockpit-project/cockpit"`?

Reply:
this method is called two ways, once with `repo=None` from main and once with `repo=...` for each external-project. If this would be cloned into c-p/c-p and run then the one call with None (which with this PR becomes c-p/c-p( is is fact the main run and not an external-project.

Reply:
Right, I see the two cases (the "main" repo and if it's cockpit, look at externals), but it feels like that should be handled at the top level, and `cockpit_tasks()` and `scan_for_pull_tasks()` would ideally not know about that. I'll send a separate comment to the place where it uses that flag.